finalPrep/modZeroHero.js

```diff
@@ -1,52 +1,88 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
+var heroName = "Kyra";
```

Comment:
I figured it out- I worked with a Kylie, but what threw me off is my wife's niece is named Kyra (eye, not key)!